machines/sns52.nix

```diff
@@ -29,13 +29,13 @@ in {
   
```

Comment:
You also need to add `virtualisation.docker.enable = true;`

(see `https://github.com/princeton-sns/cluster/blob/main/machines/sns33.nix`)

Reply:
Hmm it looks like it's already set to True? (line 22)

Reply:
Oops! Sorry